src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -679,10 +843,11 @@ impl LinearActuatedRigidBodyOnHingeAxis {
     }
 
     /// Indicates correct direction of the rigid body when an actuator would be extending or compressing.
-    /// If compressing actuator would give a rising rigid body angle, sets TRUE
+    /// If extending actuator would give a rising rigid body angle, sets TRUE
```

Comment:
I don't think this change was intentional. 🙂

Reply:
Oh it was! I thought it was easier to read if using extending in both case so we don't get lost reading it. I might have been wrong considering this remark :O
Logic was also reverted from what I was expecting in documented convention: positive force when extending